src/litdata/streaming/reader.py

```diff
@@ -432,17 +435,17 @@ def __del__(self) -> None:
             self._prepare_thread = None
 
 
-def _get_folder_size(path: str) -> int:
+def _get_folder_size(path: str, config: ChunksConfig) -> int:
     """Collect the size of each files within a folder.
 
     This method is robust to file deletion races
 
     """
     size = 0
-    for dirpath, _, filenames in os.walk(str(path)):
-        for filename in filenames:
+    for filename in os.listdir(os.path.join(path, "cache_dir")):
```

Comment:
We shouldn't add cache_dir there.

Reply:
Config does seem to have the `cache_dir`, so may be we could use from there.`self._config._cache_dir`